### PR TITLE
v2.3 REST API and postman collection changes

### PR DESCRIPTION
**Pull request by manuwelakanade-GSLAB**
v2.3 REST API and postman collection changes

### CODE REVIEW

**Comment by manuwelakanade-GSLAB** [comment]
Hi @JosephPJordan : Can you please review and merge this pull request. Also edited conf.py to have 2.3 - please let me know if the version needs to be changed in any other file. Thank you.

File: docs/rest_api_ep.rst [hunk 1]
```diff
@@ -0,0 +1,274 @@
+REST API Dynamic Endpoint Attach Detach
+=======================================
+
+This section will go over the specifics of the REST APIs supported by F5 ACI ServiceCenter application for dynamic endpoint attach detach feature.
```
Comment (JosephPJordan) [inline docs/rest_api_ep.rst]: This section contains the specifics of the REST APIs supported by the F5 ACI ServiceCenter application  for the dynamic endpoint attach detach feature.

File: docs/rest_api_ep.rst [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+   Example:
+
+   https://10.107.0.24/appcenter/F5Networks/F5ACIServiceCenter/getbigiplist.json
+
+`Download the API POSTMAN collection <https://github.com/F5Networks/f5-aci-servicecenter/tree/master/api_collection>`_.
+
+Request Headers
+---------------
+
+For all the F5 ACI ServiceCenter API calls following request header parameters are required:
```
Comment (JosephPJordan) [inline docs/rest_api_ep.rst]: For all the F5 ACI ServiceCenter API calls, the following request header parameters are required:

File: docs/rest_api_ep.rst [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+
+2. Body for the POST request
+
+.. code-block:: json
+
+   <aaaUser name="apic-username" pwd="apic-password"/>
+
+3. From the result of the POST, save the token returned
+
+4. In the “Headers” section of any further REST API requests, add a key-value pair. Key name is “DevCookie”, and its value should be the token obtained in the previous step
```
Comment (JosephPJordan) [inline docs/rest_api_ep.rst]: 4. In the **Headers** section of any further REST API requests, add a key-value pair. Key name is **DevCookie**, and its value should be the token obtained in the previous step

File: docs/rest_api_2.rst [hunk 4]
```diff
@@ -0,0 +1,4156 @@
+REST API
+========
+
+This section will go over the specifies of the REST APIs supported by F5 ACI ServiceCenter application.
```
Comment (JosephPJordan) [inline docs/rest_api_2.rst]: This section contains the specifics of the REST APIs supported by the F5 ACI ServiceCenter application.

File: docs/rest_api_2.rst [hunk 5]
```diff
@@ -0,0 +1,4156 @@
...
+   Example:
+
+   https://10.107.0.24/appcenter/F5Networks/F5ACIServiceCenter/getbigiplist.json
+
+`Download the API POSTMAN collection <https://github.com/F5Networks/f5-aci-servicecenter/tree/master/api_collection>`_.
+
+Request Headers
+---------------
+
+For all the F5 ACI ServiceCenter API calls following request header parameters are required:
```
Comment (JosephPJordan) [inline docs/rest_api_2.rst]: ...API calls, the following request header...

File: docs/rest_api_2.rst [hunk 6]
```diff
@@ -0,0 +1,4156 @@
...
+
+2. Body for the POST request
+
+.. code-block:: json
+
+   <aaaUser name="apic-username" pwd="apic-password"/>
+
+3. From the result of the POST, save the token returned
+
+4. In the “Headers” section of any further REST API requests, add a key-value pair. Key name is “DevCookie”, and its value should be the token obtained in the previous step
```
Comment (JosephPJordan) [inline docs/rest_api_2.rst]: replace quotes with ** ** to bold those words

File: docs/rest_api_2.rst [hunk 7]
```diff
@@ -0,0 +1,4156 @@
...
+````````````````````
+.. code-block:: json
+   
+   Key           Value           
+   Content-Type: application/json
+   
+Input Parameters
+----------------
+
+There are multiple F5 ACI ServiceCenter APIs which require input parameters to be retrieved for the APIC Logical Device. These input parameters can be retrieved from APIC object browser known as visore.html
```
Comment (JosephPJordan) [inline docs/rest_api_2.rst]: ...can be retrieved from the APIC object browser...

File: docs/rest_api_2.rst [hunk 8]
```diff
@@ -0,0 +1,4156 @@
...
++=============+================================================================================================================================+=========================================================================================================================================================+
+| ldev        | 1. Go to APIC GUI → Tenant→ Services→ L4-L7→ Devices→ <Your L4-L7 Device>                                                      |     uni/tn-Sample\_2/lDevVip-f5-gs                                                                                                                      |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 2. Right click on the <Your L4-L7 Device> and click on “Open in Object Store Browser”                                          |                                                                                                                                                         |
```
Comment (JosephPJordan) [inline docs/rest_api_2.rst]: remove both "on" and replace quotes with ** **, ie Right click the <your L4..., and click **Open in Object...

File: docs/rest_api_2.rst [hunk 9]
```diff
@@ -0,0 +1,4156 @@
...
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 3. Use the dn property from the object browser for input parameter “ldev”.                                                     |                                                                                                                                                         |
```
Comment (JosephPJordan) [inline docs/rest_api_2.rst]: the input parameter

File: docs/rest_api_2.rst [hunk 10]
```diff
@@ -0,0 +1,4156 @@
...
++-------------+--------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+
+| lif         | 1. Go to APIC GUI → Tenant→ Services→ L4-L7→ Devices→ <Your L4-L7 Device> → Cluster Interfaces→ <Your Interface of Interest>   |     uni/tn-Sample\_2/lDevVip-f5-gs/lIf-external                                                                                                         |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 2. Right click on the logical interface and click on “Open in Object Store Browser”                                            |                                                                                                                                                         |
```
Comment (JosephPJordan) [inline docs/rest_api_2.rst]: remove "on" in both places and replace "" with ** **

File: docs/rest_api_2.rst [hunk 11]
```diff
@@ -0,0 +1,4156 @@
...
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 1. Go to APIC GUI → Tenant→ Services→ L4-L7→ Devices→ <Your L4-L7 Device> → Cluster Interfaces→ <Your Interface of Interest>   |                                                                                                                                                         |
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 2. Right click on the logical interface and click on “Open in Object Store Browser”.                                           |                                                                                                                                                         |
```
Comment (JosephPJordan) [inline docs/rest_api_2.rst]: remove "on"s and bold instead of quotes

File: docs/rest_api_2.rst [hunk 12]
```diff
@@ -0,0 +1,4156 @@
...
+|             |                                                                                                                                |                                                                                                                                                         |
+|             | 3. Click on > arrow of the dn property to see it’s children                                                                    |                                                                                                                                                         |
```
Comment (JosephPJordan) [inline docs/rest_api_2.rst]: Click the > arrow of the dn property to see its children.

File: docs/rest_api_ep.rst (hunk 1 above)
Comment (manuwelakanade-GSLAB) [reply docs/rest_api_ep.rst]: Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

File: docs/rest_api_ep.rst (hunk 2 above)
Comment (manuwelakanade-GSLAB) [reply docs/rest_api_ep.rst]: Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

File: docs/rest_api_ep.rst (hunk 3 above)
Comment (manuwelakanade-GSLAB) [reply docs/rest_api_ep.rst]: Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

File: docs/rest_api_2.rst (hunk 4 above)
Comment (manuwelakanade-GSLAB) [reply docs/rest_api_2.rst]: Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

File: docs/rest_api_2.rst (hunk 5 above)
Comment (manuwelakanade-GSLAB) [reply docs/rest_api_2.rst]: Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

File: docs/rest_api_2.rst (hunk 6 above)
Comment (manuwelakanade-GSLAB) [reply docs/rest_api_2.rst]: Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

File: docs/rest_api_2.rst (hunk 7 above)
Comment (manuwelakanade-GSLAB) [reply docs/rest_api_2.rst]: Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

File: docs/rest_api_2.rst (hunk 8 above)
Comment (manuwelakanade-GSLAB) [reply docs/rest_api_2.rst]: Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

File: docs/rest_api_2.rst (hunk 9 above)
Comment (manuwelakanade-GSLAB) [reply docs/rest_api_2.rst]: Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

File: docs/rest_api_2.rst (hunk 10 above)
Comment (manuwelakanade-GSLAB) [reply docs/rest_api_2.rst]: Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

File: docs/rest_api_2.rst (hunk 11 above)
Comment (manuwelakanade-GSLAB) [reply docs/rest_api_2.rst]: Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

File: docs/rest_api_2.rst (hunk 12 above)
Comment (manuwelakanade-GSLAB) [reply docs/rest_api_2.rst]: Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)

**Comment by manuwelakanade-GSLAB** [comment]
Hi @JosephPJordan : Thanks for the suggesting the changes. I have addressed all the comments in rest_api.rst file because that is the final name of the file. Please let me know if you have any further suggestions.

**Pull request merged**